plugin/crazyegg.xml

```diff
@@ -38,4 +38,7 @@
             </fieldset>
         </fields>
     </config>
+    <updateservers>
+        <server type="extension">https://ce-development.s3.amazonaws.com/joomla/updates.xml</server>
```

Comment:
may be use crazyeggcdn.com? @imtall what do you suggest for hosting public files like these

Reply:
I think it's a good idea to use `crazyeggcdn.com` here.

@fatum - can you you set up `plugins.crazyeggcdn.com` and have that point to a new S3 bucket that contains `/joomla/updates.xml`?

Reply:
updated 👍

Reply:
Looks good to me, thanks!